main/acle.md

```diff
@@ -9382,6 +9416,2341 @@ are named after. All of the functions have external linkage.
+  //   _za32[_u16]
+  //   _za64[_s16] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  //   _za64[_u16] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svdot_lane_za32[_bf16]_vg1x4(uint32_t slice_base, uint64_t slice_offset,
+                                    svbfloat16x4_t zn, svbfloat16_t zm,
+                                    uint64_t imm_idx);
+  
+  
+  
+  __attribute__((arm_streaming, arm_shared_za))
+  void svsudot_lane_za32[_s8]_vg1x2(uint32_t slice_base, uint64_t slice_offset,
+                                    svint8x2_t zn, svuint8_t zm,
+                                    uint64_t imm_idx);
+  
+  
+  
+  __attribute__((arm_streaming, arm_shared_za))
+  void svsudot_lane_za32[_s8]_vg1x4(uint32_t slice_base, uint64_t slice_offset,
+                                    svint8x4_t zn, svuint8_t zm,
+                                    uint64_t imm_idx);
+  
+  
+  
+  __attribute__((arm_streaming, arm_shared_za))
+  void svusdot_lane_za32[_u8]_vg1x2(uint32_t slice_base, uint64_t slice_offset,
+                                    svuint8x2_t zn, svint8_t zm,
+                                    uint64_t imm_idx);
+  
+  
+  
+  __attribute__((arm_streaming, arm_shared_za))
+  void svusdot_lane_za32[_u8]_vg1x4(uint32_t slice_base, uint64_t slice_offset,
+                                    svuint8x4_t zn, svint8_t zm,
+                                    uint64_t imm_idx);
+  
+  ```
+
+#### FVDOT, BFVDOT, SUVDOT, USVDOT, SVDOT, UVDOT
+
+Multi-vector vertical dot-product by indexed element.
+
+``` c
+  
+  __attribute__((arm_streaming, arm_shared_za))
+  void svsuvdot_lane_za32[_s8]_vg1x4(uint32_t slice_base, uint64_t slice_offset,
+                                     svint8x4_t zn, svuint8_t zm,
+                                     uint64_t imm_idx);
+  
+  
+  
+  __attribute__((arm_streaming, arm_shared_za))
+  void svusvdot_lane_za32[_u8]_vg1x4(uint32_t slice_base, uint64_t slice_offset,
+                                     svuint8x4_t zn, svint8_t zm,
+                                     uint64_t imm_idx);
+  
+  
+  // Variants are also available for _za32[_f16], _za32[_s16] and _za32[_u16]
+  __attribute__((arm_streaming, arm_shared_za))
+  void svvdot_lane_za32[_bf16]_vg1x2(uint32_t slice_base, uint64_t slice_offset,
+                                     svbfloat16x2_t zn, svbfloat16_t zm,
+                                     uint64_t imm_idx);
+  
+  
+  // Variants are available for:
+  //   _za32[_s8]
+  //   _za32[_u8]
+  //   _za64[_s16] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  //   _za64[_u16] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svvdot_lane_za32[_s8]_vg1x4(uint32_t slice_base, uint64_t slice_offset,
+                                   svint8x4_t zn, svint8_t zm, uint64_t imm_idx);
+  
+  ```
+
+#### UMOPA, SMOPA, UMOPS, SMOPS
+
+Integer sum of outer products and accumulate/subtract (2-way and 4-way)
```

Comment:
I think this should be 

Integer sum of outer products and accumulate/subtract (2-way)

because the 4-way variants were added as part of SME1.